ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -606,3 +606,28 @@ def polymul(a1, a2, *, trim_leading_zeros=False):
     if len(a2) == 0:
         a2 = ivy.asarray([0], dtype=a2.dtype)
     return convolve(a1, a2, mode="full")
+
+@to_ivy_arrays_and_back
+def product(a,
+            *,
+            axis=None,
+            dtype=None,
+            keepdims=False,
+            initial=None,
+            where=None,
+            promote_integer=True,
```

Comment:
The original `jax` function has the argument `promote_integers`

Reply:
I have changed `promote_integer` to `promote_integers`